osu.Desktop/Overlays/VersionManager.cs

```diff
@@ -55,23 +57,30 @@ private void load(OsuColour colours, TextureStore textures, OsuGameBase game)
                                 },
                             }
                         },
-                        new OsuSpriteText
-                        {
-                            Anchor = Anchor.TopCentre,
-                            Origin = Anchor.TopCentre,
-                            Font = OsuFont.Numeric.With(size: 12),
-                            Colour = colours.Yellow,
-                            Text = @"Development Build"
-                        },
-                        new Sprite
-                        {
-                            Anchor = Anchor.TopCentre,
-                            Origin = Anchor.TopCentre,
-                            Texture = textures.Get(@"Menu/dev-build-footer"),
-                        },
                     }
                 }
             };
+
+            if (DebugUtils.IsDebugBuild)
```

Comment:
Thoughts about using `OsuGameBase.IsDeployedBuild` instead? I know you don't really run local releases - I sometimes do, but am not likely to get confused - but just throwing that out there as a potentially better suggestion.

Reply:
That's the variable I was looking for.